File: dev-docs/snippets/analytics/overrides.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Override the `plan` and `app_version` fields stamped on every subsequent
// analytics event. Pass `null` to clear an override.
export function overrides(client: JsBaoClient) {
  // #region example
  client.analytics.setPlanOverride("pro");
  client.analytics.setAppVersionOverride("2.4.1");
  // #endregion example
}

File: dev-docs/snippets/analytics/overrides.swift
import JsBaoClient

// Override the `plan` and `app_version` fields stamped on every subsequent
// analytics event. Pass `nil` to clear an override.
func overrides(client: JsBaoClient) {
  // #region example
  client.analytics.setPlanOverride("pro")
  client.analytics.setAppVersionOverride("2.4.1")
  // #endregion example
}
